foundation/foundation-upgrade/src/main/java/com/boozallen/aissemble/upgrade/migration/HelmChartsNameMigration.java

```diff
@@ -0,0 +1,80 @@
+package com.boozallen.aissemble.upgrade.migration;
+
+/*-
+ * #%L
+ * aiSSEMBLE::Foundation::Upgrade
+ * %%
+ * Copyright (C) 2021 Booz Allen
+ * %%
+ * This software package is licensed under the Booz Allen Public License. All Rights Reserved.
+ * #L%
+ */
+
+import com.boozallen.aissemble.upgrade.util.FileUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import java.io.File;
+import java.io.IOException;
+import java.util.ArrayList;
+import static com.boozallen.aissemble.upgrade.util.FileUtils.getRegExCaptureGroups;
+
```

Comment:
A: I know this is a little self explanatory, but a little context on the class comment can really help in like 6 months

Reply:
@d-ryan-ashcraft good point! added descriptions and made adjustments per your comments below.